.releaserc.yaml

```diff
@@ -8,7 +8,14 @@ plugins:
   - - "@semantic-release/exec"
     - prepareCmd: "./deploy.sh ${nextRelease.version}"
   - - "@semantic-release/git"
-    - assets: ["pubspec.yaml"]
+    - assets:
+        - pubspec.yaml
       message: "chore(release): ${nextRelease.version}"
 branches:
-  - master
+  - name: master
+    channel: alpha
+    prerelease: alpha
```

Comment:
wait, master is alpha? why not master be stable? or is the idea to keep things going into master?

Reply:
The idea would be to keep things going in master

Reply:
So it will go as such 

`master -> beta -> stable (upstream is beta)`

and SR will handle the versioning and the fastlane just needs to be edited to push to each lane based on the branch it's running on.